config/_default/params.yaml

```diff
@@ -0,0 +1,154 @@
+  license: MIT
+
+# github information
+#  github_repo:  url to repo of web site
+#  github_branch:  main branch of repo
+#  github_subdir:  unused by Interlisp.org
+#  github_project_repo: added by Interlisp.org points to repo used
+#                       for issue reporting
+#  
+github_repo: https://github.com/interlisp/Interlisp.github.io
+github_branch: main
+github_subdir: 
+github_project_repo: https://github.com/interlisp/medley
+
+# Google custom seach engine configuration
+#  gcs_engine_id:  search engine 
+gcs_engine_id: 33ef4cbe0703b4f3a
+
+# Footer Links
+# 
+# user:  End user relevant links. These will show up on 
+#        left side of footer and in the community page if you have one.
+#   name: displayed when moused over
+#   url:  url to use when selected
+#   desc: description
+#   icon:  icon to display
+links:
+  user:
+  - name: "User mailing list"
+    url: "mailto:Interlisp@googlegroups.com"
+    icon: "fa fa-envelope"
+    desc:  "Discussion and help from your fellow users"
+  - name: "Twitter"
+    url: "https://twitter.com/interlisp8"
+    icon: "fab fa-twitter"
+    desc: "Follow us on Twitter to get the latest news!"
+# - name: "Stack Overflow"
+#   url: "https://stackoverflow.com/questions/tagged/graphviz"
+#   icon: "fab fa-stack-overflow"
+#   desc: "Practical questions and curated answers"
+
+# developer:  Developer relevant links. These will show up on right 
+#             side of footer and in the community page if you have one.
+  developer:
+  - name: "On GitHub"
+    url: "https://github.com/interlisp"
+    icon: "fab fa-github"
+    desc: "Development takes place here!"
+# - name: "Slack"
+#   url: "https://example.org/slack"
+#   icon: "fab fa-slack"
+#   desc: "Chat with other project developers"
+# - name: "Developer mailing list"
+#   url: "https://example.org/mail"
+#   icon: "fa fa-envelope"
+#   desc: "Discuss development issues around the project"
+
+taxonomies:
+# Nothing defined
+
+# User Interface Configuration options
+#
+# feedback:
+#   Adds a H2 section titled "Feedback" to the bottom of each doc. The responses 
+#   are sent to Google Analytics as events.  This feature depends 
+#   on [services.googleAnalytics] and will be disabled if 
+#   "services.googleAnalytics.id" is not set.
+#
+#   If you want this feature, but occasionally need to remove the 
+#   "Feedback" section from a single page,
+#   add "hide_feedback: true" to the page's front matter.
+#    enable:  boolean Turn feedback
+#    yes: text to display with yes is selected
+#    no: text to display when no is selected
+
+ui:
+  feedback:
+    enable: true
+    yes: 'Glad to hear it! Please <a href="https://github.com/Interlisp/medley/issues/new?template=documentation.md&projects=Interlisp/medley/5">tell us how we can improve</a>.'
+    no: 'Sorry to hear that. Please <a href="https://github.com/Interlisp/medley/issues/new?template=documentation.md&projects=Interlisp/medley/5">tell us how we can improve</a>.'
+
```

Comment:
I like the idea of per-page asking for feedback, but I havent seen any.
Is it how we're asking?

Reply:
Possibly - I really have no idea how much feedback other sites get from this type of setup.  

But, I can imagine that with longer documents you have an issue with users even getting to the feedback button.  If, the information isn't useful, they'll be long gone before they get to it.